.rubocop.yml

```diff
@@ -1,5 +1,6 @@
 ---
 AllCops:
+  TargetRubyVersion: 2.4
```

Comment:
Added this to pin the ruby version rubocop was targeting. I was seeing errors in Travis due to default target ruby version that weren't showing up in my local environment and vice versa. I don't have an attachment to 2.4, just want something